### PR TITLE
removing gas fraction

### PR DESCRIPTION
**Pull request by berkserbet**
### Why:
* Discounts gas from total loadshapes.

### What:
* Working implementation, in the future gas fractions should be read as other climate zones.

### Checklist
- [x] Finished My Changes
- [ ] Automated Unit Tests

### CODE REVIEW

File: pipelines/rbsa/tasks/discount_gas.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        for city in city_list:
+
+            city_df = self.df.loc[self.df.target == city]
+
+            zone = self.projection_locations['cities'][city]
+            electric_percentage = self.gas_fraction['electrification'][zone]   
+
+            # add gas fraction
+            for enduse in electric_percentage.keys():
+                city_df[enduse] = city_df[enduse]*city_df[enduse]
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/discount_gas.py]: whitespace around `*`

File: pipelines/rbsa/rbsa.py [hunk 2]
```diff
@@ -6,7 +6,8 @@
     undiscount_gas, 
     index_heatcool, 
     normalize_totals,
-    zipcode_correlation
+    zipcode_correlation, 
+    discount_gas
```
Comment (jongoncalves) [inline pipelines/rbsa/rbsa.py]: Make sure when these are GTG and you start merging into master, mitigate the merge conflicts and the ordering of tasks.

File: pipelines/rbsa/tasks/discount_gas.py (hunk 1 above)
Comment (berkserbet) [reply pipelines/rbsa/tasks/discount_gas.py]: fixed

File: pipelines/rbsa/rbsa.py (hunk 2 above)
Comment (berkserbet) [reply pipelines/rbsa/rbsa.py]: I'll make sure to pull master and fix after every step

**Pull request merged**